packages/maintenance-app/frontend/package.json

```
@@ -16,7 +16,7 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
-    "@veriphi/veto-ui": "^0.2.0",
+    "@veriphi/veto-ui": "^0.3.0",
```

Should it use the same version as veto-app?
```suggestion
    "@veriphi/veto-ui": "^0.4.0",
```